core/dbt/include/global_project/macros/adapters/common.sql

```diff
@@ -60,6 +60,17 @@
   );
 {% endmacro %}
 
+{% macro create_or_replace_table_as(relation, sql) -%}
```

Comment:
I think we can remove these macros and just add `... or replace ...` to the Snowflake implementation of [create_table_as](https://github.com/fishtown-analytics/dbt/blob/32f74b60ef4e650d339ea59b98babb27206d433d/plugins/snowflake/dbt/include/snowflake/macros/adapters.sql#L1-L16). We do the same for [snowflake__create_view_as](https://github.com/fishtown-analytics/dbt/blob/32f74b60ef4e650d339ea59b98babb27206d433d/plugins/snowflake/dbt/include/snowflake/macros/adapters.sql#L18-L22) and it seems to work ok!

Reply:
yes! done